modules/ossm-configuring-jaeger-existing-v1x.adoc

```diff
@@ -0,0 +1,94 @@
+
+* Supply the correct jaeger-collector endpoint to Mixer by setting `spec.istio.global.tracer.zipkin.address` to the hostname and port of your jaeger-collector service. The hostname of the service is usually `<jaeger-instance-name>-collector.<namespace>.svc`.
+
+* Supply the correct jaeger-query endpoint to Kiali for gathering traces by setting `spec.istio.kiali.jaegerInClusterURL` to the hostname of your jaeger-query service - the port is normally not required, as it uses 443 by default. The hostname of the service is usually  `<jaeger-instance-name>-query.<namespace>.svc`.
+
+* Supply the dashboard URL of your Jaeger instance to Kiali to enable accessing Jaeger through the Kiali console. You can retrieve the URL from the OpenShift route that is created by the Jaeger Operator. If your Jaeger resource is called `external-jaeger` and resides in the `istio-system` project, you can retrieve the route using the following command:
++
+[source,terminal]
+----
+$ oc get route -n istio-system external-jaeger
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   HOST/PORT                                     PATH   SERVICES               [...]
+external-jaeger        external-jaeger-istio-system.apps.test        external-jaeger-query  [...]
+----
++
+The value under `HOST/PORT` is the externally accessible URL of the Jaeger dashboard.
+
+
+.Example Jaeger resource
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "external-jaeger"
+  # Deploy to the Control Plane Namespace
+  namespace: istio-system
+spec:
+  # Set Up Authentication
+  ingress:
+    enabled: true
+    security: oauth-proxy
+    openshift:
+      # This limits user access to the Jaeger instance to users who have access
+      # to the control plane namespace. Make sure to set the correct namespace here
+      sar: '{"namespace": "istio-system", "resource": "pods", "verb": "get"}'
+      htpasswdFile: /etc/proxy/htpasswd/auth
+
+  volumeMounts:
+  - name: secret-htpasswd
+    mountPath: /etc/proxy/htpasswd
+  volumes:
+  - name: secret-htpasswd
+    secret:
+      secretName: htpasswd
+
+----
+
+The following `ServiceMeshControlPlane` example assumes that you have deployed Jaeger using the Jaeger Operator and the example Jaeger resource.
+
+.Example `ServiceMeshControlPlane` with external Jaeger
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+metadata:
+  name: external-jaeger
+  namespace: istio-system
+spec:
+  version: v1.1
+  istio:
+    tracing:
+      # Disable Jaeger deployment by service mesh operator
+      enabled: false
+    global:
+      tracer:
+        zipkin:
+          # Set Endpoint for Trace Collection
+          address: external-jaeger-collector.istio-system.svc:9411
+    kiali:
+      # Set Jaeger dashboard URL
+      dashboard:
+        jaegerURL: https://external-jaeger-istio-system.apps.test
```

Comment:
Are you sure this is the correct URL?

Reply:
The URL in the Maistra docs looked like it was from internal testing.  (https://maistra.io/docs/monitoring_and_tracing/using-existing-jaeger/)   I assumed the URL would be different for each user.  Was that an incorrect assumption to make?

Reply:
I think you would need to include some instructions, such as in the "Set Jaeger dashboard URL" from the referenced doc you mentioned.

Reply:
>I think you would need to include some instructions...

Like line 20 - 34?  😀